examples/cfd/swe_distributed_gnn/train.py

```diff
@@ -0,0 +1,640 @@
+# coding=utf-8
```

Comment:
Need to include # ignore_header_test to pass the license test.
Also, would be great to mention that this example was adopted from the torch harmonics package, to give credit to the developers.

Reply:
done